Dockerfile

```diff
@@ -11,6 +11,8 @@ COPY package.json package-lock.json ./
 RUN npm ci
```

Comment:
Why we need copy it twice?

Reply:
The first is to install dependencies to build, and the second is to install runtime environment, `npm start` use the `probot` to run.

Reply:
Maybe we can build the project in host and just run in container?
![image](https://user-images.githubusercontent.com/5086433/106259085-b39db280-6259-11eb-8067-4067727ba68d.png)

Reply:
But I think we only need copy these files once.

Reply:
You can refer to https://github.com/ti-community-infra/ti-community-bot/blob/master/Dockerfile

Reply:
But if we can remove the dependency of development environment from the image, can we reduce the space occupied by image

Reply:
If we compile on the GitHub action machine, we can also make good use of the GitHub cache.

Reply:
```
FROM node:12-slim

WORKDIR /usr/src/app

COPY package.json package-lock.json ./

RUN npm ci --production
RUN npm cache clean --force

COPY ./lib lib
COPY ormconfig.js ./

ENV NODE_ENV="production"
EXPOSE 3000

CMD [ "npm", "start" ]
```